dlgr/griduniverse/experiment.py

```diff
@@ -94,17 +94,16 @@
     'donation_group': bool,
     'donation_ingroup': bool,
     'donation_public': bool,
-    'num_food': int,
-    'respawn_food': bool,
-    'food_visible': bool,
-    'food_reward': int,
-    'food_pg_multiplier': float,
-    'food_growth_rate': float,
-    'food_maturation_speed': float,
-    'food_maturation_threshold': float,
-    'food_planting': bool,
-    'food_planting_cost': int,
-    'food_probability_distribution': unicode,
+    'object_count': int,
```

Comment:
@cguardia I don't think any of these should be in the experiment config, they should be object properties in the YAML and don't need to be registered here.

Reply:
I'm going to propose a generic `plantable` property unless someone can think of a better word. Seems generic enough (eg: you can "plant evidence", etc.)

Reply:
Sure, for now we'll only be using it for food, so it doesn't much matter I think.